package.json

```diff
@@ -69,8 +69,8 @@
   },
   "peerDependencies": {
     "aws-cdk": "2.23.0",
```

Comment:
Fancy addressing this https://github.com/guardian/cdk/pull/1276#issuecomment-1129061289 too?